TestSamples/SwiftUITestSample/Maestro/corruptEnvelope.yaml

```diff
@@ -0,0 +1,9 @@
+appId: io.sentry.tests.SwiftUITestSample
+---
+- launchApp
+- tapOn: "Write Corrupted Envelope"
+- tapOn: "Close SDK"
+- tapOn: "Write Corrupted Envelope"
```

Comment:
`m`: What drove you to close the SDK between writing corrupted envelopes?

Reply:
The test first write corrupted envelope while the SDK is running, so what happens is this gets picked up and processed and deleted pretty much immediately. This ensures that the SDK doesn't crash while running.

After that I close the SDK to ensure the next written corrupted envelope will be present during the SDK start up.

Reply:
Added to the test file.

Reply:
That seems like actually two test cases. Anyways it would make sense to add this info as a comment.